packages/devtools_app/lib/src/screens/memory/panes/allocation_profile/allocation_profile_table_view.dart

```diff
@@ -398,10 +399,19 @@ class _ExportAllocationProfileButton extends StatelessWidget {
           tooltip: 'Download allocation profile data in CSV format',
           onPressed: currentAllocationProfile == null
               ? null
-              : () {
-                  final file = allocationProfileController
+              : () async {
+                  final loc = await allocationProfileController
```

Comment:
nit: use 'location' instead of abbreviating (I initially thought "loc" = "line of code")